src/v2/controllers/FormController.js

```diff
@@ -217,21 +214,36 @@ export default Controller.extend({
       });
   },
 
+  transformIdentifier(formName, model) {
```

Comment:
nit: Since we're refactoring - how about extracting out this method to a `util` file?

Reply:
could be (would need another parameter `this.settings`).

Reply:
Ah - missed this! We can always approach this later.